tests/profiles-enabled/conftest.py

```diff
@@ -161,6 +205,31 @@ def mocked_not_moderator(http_pretty, shipment):
                              body=json.dumps({'bad': 'bad'}), status=status.HTTP_403_FORBIDDEN)
 
 
+@pytest.fixture
+def mock_non_wallet_owner_calls(http_pretty):
+    wallet_url = f'{test_settings.PROFILES_URL}/api/v1/wallet/{SHIPPER_ID}/'
+    storage_credentials_url = f'{test_settings.PROFILES_URL}/api/v1/storage_credentials/{SHIPPER_ID}/'
+    http_pretty.register_uri(http_pretty.GET, wallet_url, status=status.HTTP_404_NOT_FOUND)
+    http_pretty.register_uri(http_pretty.GET, wallet_url, status=status.HTTP_404_NOT_FOUND)
```

Comment:
I do not understand the need of these two lines

Reply:
This is actually unecessary here. Technically we only really need the first successful check as it would not even hit the next two checks.

Reply:
This is necessary as it will check the same endpoint twice, and therefore needs to be mocked twice. If only one failure is mocked then it can fail to fail.